src/plugins/betterSessions/index.tsx

```diff
@@ -0,0 +1,221 @@
+        ];
+
+        // Show a "NEW" badge if the session is seen for the first time
+        if (savedSession == null || savedSession.isNew) {
+            children.splice(1, 0,
+                <div
+                    className="vc-plugins-badge"
+                    style={{
+                        backgroundColor: "#ED4245",
+                        marginLeft: "2px"
+                    }}
+                >
+                    NEW
+                </div>
+            );
+        }
+
+        return children;
+    },
+
+    renderTimestamp({ session }: SessionInfo, timeLabel: string) {
+        return (
+            <Tooltip text={session.approx_last_used_time.toLocaleString()} tooltipClassName={TimestampClasses.timestampTooltip}>
+                {props => (
+                    <span {...props} className={TimestampClasses.timestamp}>
+                        {timeLabel}
+                    </span>
+                )}
+            </Tooltip>
+        );
+    },
+
+    renderRenameButton({ session }: SessionInfo, state: [string, React.Dispatch<React.SetStateAction<string>>]) {
+        return (
+            <Button
+                look={Button.Looks.LINK}
+                color={Button.Colors.LINK}
+                size={Button.Sizes.NONE}
+                style={{
+                    paddingTop: "0px",
+                    paddingBottom: "0px",
+                    top: "-2px"
+                }}
+                onClick={() =>
+                    openModal(props => (
+                        <RenameModal
+                            props={props}
+                            session={session}
+                            state={state}
+                        />
+                    ))
+                }
+            >
+                Rename
+            </Button>
+        );
+    },
+
+    renderIcon({ session }: SessionInfo) {
+        const PlatformIcon = GetPlatformIcon(session.client_info.platform);
+
+        return (
+            <div
+                className={SessionIconClasses.sessionIcon}
+                style={{ backgroundColor: GetOsColor(session.client_info.os) }}
+            >
+                <PlatformIcon width={32} height={32} />
+            </div>
+        );
+    },
+
+    async checkNewSessions() {
+        const data = await RestAPI.get({
+            url: "/auth/sessions"
+        });
+
+        const newSessions = data.body.user_sessions.filter((session: SessionInfo["session"]) => !savedSessionsCache.has(session.id_hash));
+        for (const session of newSessions) {
+            savedSessionsCache.set(session.id_hash, { name: "", isNew: true });
+
+            showNotification({
+                title: "BetterSessions",
+                body: `New session:\n${session.client_info.os} · ${session.client_info.platform} · ${session.client_info.location}`,
+                permanent: true,
+                onClick: () => UserSettingsAccountActionCreators.open(Constants.UserSettingsSections.SESSIONS)
```

Comment:
i would just hardcode these constants since they're never gonna change. It's more likely for this find to break, than for the constant value to change. Plus, it costs like 2-20ms to search for it (which adds up)
```suggestion
                onClick: () => UserSettingsAccountActionCreators.open("Sessions")
```